manager/middlewares/authentication.go

```diff
@@ -1,13 +1,21 @@
 package middlewares
 
 import (
+	"app/base/database"
+	"app/base/models"
 	"app/base/utils"
 	"github.com/gin-gonic/gin"
 	"net/http"
+	"sync"
 )
 
 const KeyAccount = "account"
 
+var AccountIDCache = struct {
```

Comment:
Don't you rather use lib for cache object? https://github.com/hashicorp/golang-lru

Reply:
No need, since cache is never cleaned up. I'd rather leave it as is, than pull in another dependency